### ISSUE TITLE
Add a link to a place to report issues with projects

### ISSUE THREAD

**Issue by mrobinson** [Igalia/phpreport#482]
It might be good to have a link specifically for reporting issues with projects or to request the creation of a new project. This would make it clearer what to do when a project doesn't exist yet and prevent people from pinging directly when there is something missing.

This would probably look something like a link that says "Report missing or misnamed projects here."

**Comment by jaragunde** [comment]
The link to the bug tracker you mention was manually added to [web/include/menubar.php](https://github.com/Igalia/phpreport/blob/master/web/include/menubar.php). It would be great for this to be configurable for every deployment, from the configuration file. We could define the feature as follows:
* Add an entry to `config.php` called ISSUE_TRACKER_URLS, of array type, listing any number of URL strings.
* Add an entry to `config.php` called ISSUE_TRACKER_LINKS_TEXT, of array type, listing any number of text strings matching the entries from the previous field.
* Let `menubar.php` add as many links as listed in ISSUE_TRACKER_URLS to the top-right menu, using the corresponding ISSUE_TRACKER_LINKS_TEXT for the link text.

A more ambitious approach would allow us to configure help/support messages or links in various places of the UI.

**Comment by jaragunde** [comment]
Closed with pull request #486:

```
commit d0c7a8d7d25388192df98d94b03eb8443378f3dd
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Sat Jan 30 21:33:35 2021 +0100

    [#482] Add configuration to link an issue tracker from the menu.
    
    Added ISSUE_TRACKER_LINKS_URL and ISSUE_TRACKER_LINKS_TEXT to
    config.php, which allow adding links to the menu with custom text.

```
